deploy_hpo.sh

```diff
@@ -20,31 +20,43 @@ SCRIPTS_DIR="${ROOT_DIR}/scripts"
 HPO_REPO="kruize/hpo"
 HPO_VERSION=$(grep -a -m 1 "HPO_VERSION" ${ROOT_DIR}/version.py | cut -d= -f2)
 HPO_VERSION=$(sed -e 's/^"//' -e 's/"$//' <<<"$HPO_VERSION")
-echo
-echo "Using version: ${HPO_VERSION}"
-HPO_CONTAINER_IMAGE=${HPO_REPO}:${HPO_VERSION}
+
+HPO_SA_MANIFEST="manifests/hpo-operator-sa.yaml"
```

Comment:
Should we have the oc login done automatically, by taking the OCP server & credential details as cmdline options to deploy script?